core/db/entities.py

```diff
@@ -41,7 +41,8 @@ class Submission(BaseModel):
 
     __tablename__ = "submission_dim"
 
-    submission_id = sqla.Column(sqla.String(), nullable=False, unique=True)
+    # TODO: Change submission ID from an int value to a string that includes programme or fund type?
+    submission_id = sqla.Column(sqla.Integer(), nullable=False, unique=True)
```

Comment:
Should this now be string again?

Reply:
symptom of an old rebase - dropped the commit now and this is gone